ui/src/components/VolumeOverviewTab.js

```diff
@@ -192,6 +193,21 @@ const VolumeDetailCard = (props) => {
           data-cy="delete_volume_button"
         />
       </VolumeTitleSection>
+      {/*TODO: To be replaced by new <Box></Box> component*/}
+      {!isVolumeUsageRetrievable && (
+        <div style={{ width: '48rem', paddingLeft: '20px' }}>
+          <Banner
+            variant="warning"
+            icon={<i className="fas fa-exclamation-triangle" />}
+            title={intl.formatMessage({
+              id: 'prometheus_unavailable',
```

Comment:
nit: I'm not sure about this "Prometheus is unavailable" message. To me it should likely be reworded to "Monitoring information unavailable". Indeed the message could be a bit misleading here, prometheus is available but it failed to scrape the metrics from the exporter due to a slaw platform, so to avoid any confusion I would not explicitely say that prometheus is not available.